### ISSUE TITLE
Set resolution

### ISSUE THREAD

**Issue by Scorpion2185** [minetest/minetest#11626]
```
Debian 11
Minetest 5.3.0+repack-2.1
```
It seems that you cannot set the resolution.
The fullscreen works bad, it doesn't use all the space, and reduces my resolution: 3840x2160.

**Comment by rubenwardy** [comment]
`width` and `height` in All Settings will set the resolution

I do not recommend using the official fullscreen setting, it doesn't work. Instead, you should use the fullscreen option in your distro. Check your distro and DE for this - on XFCE it's alt+f11

**Comment by sfan5** [comment]
Fullscreen issues were tracked in #4921 and have already been solved in 5.5-dev, closing.

**Comment by Scorpion2185** [comment]
I am on GNOME right now and some apps don't have fullscreen, but if I uncheck "Autosave Screen Size" I can set the height that is always reduced by the window title bar, the top panel and the dock.

And if I use fullscreen with the resolution it works fine. 
On the main menu no quit button you have to use ESC.

**Comment by Scorpion2185** [comment]
I am on GNOME right now and some apps don't have fullscreen, but if I uncheck "Autosave Screen Size" I can set the height that is always reduced by the window title bar, the top panel and the dock.

And if I use fullscreen with the resolution of the screen it works fine.
On the main menu no quit button you have to use ESC.

**Comment by Scorpion2185** [comment]
If I uncheck "Autosave Screen Size" I can set the height that is always reduced by a panel that never autohides.
And if I use fullscreen with the resolution of the screen it works fine.
But I cannot minimize minetest. 

While if I use my DE fullscreen, I had to set a key on openbox, I can minimize it but every time that I launch it I have to toggle fullscreen, because of the panel (I checked "Autosave Screen Size").
